### ISSUE TITLE
Rate limits

### ISSUE THREAD

**Issue by robingroppe** [CZ-NIC/knot-resolver#90]
Hi,
Is it possible to implement response rate limiting (rrl) and possibly recursive client rate limiting (rcrl)?

**Comment by vcunat** [comment]
No, currently we don't have that.  Linux iptables/nftables can do such limiting, at least for some use cases.

**Comment by vcunat** [comment]
There's a blog post describing work in progress on that, in case that's also interesting: https://en.blog.nic.cz/2024/07/15/knot-resolver-6-news-dos-protection-operators-overview/
